examples_tests/51.WavesSimulation/WaveSimApp.cpp

```diff
@@ -98,9 +100,10 @@ bool WaveSimApp::CreateComputePipelines()
 {
 	enum class EPipeline
 	{
-		SPECTRUM_RANDOMIZE,
-		ANIMATE_STAGE_1,
-		ANIMATE_STAGE_2,
+		RANDOMIZE_SPECTRUM,
+		ANIMATE_SPECTRUM,
+		IFFT_STAGE_1,
```

Comment:
you can merge these two together

Reply:
Yeah it used to be like that, but i couldn't find a good way to sync buffer filling and ifft. The only safe way to do it seems to be generating the animated spectrum in nbl_glsl_ext_FFT_getData, but i dont think this function is called once for every position, so it might be pretty costy

Reply:
make a global array variable (its per thread), read and animate the spectrum into that in `main()` before the FFT, then make `nbl_glsl_ext_FFT_getData` return that global